project/AppDependencies.scala

```diff
@@ -1,45 +1,30 @@
 import sbt._
+import play.core.PlayVersion
 
 object AppDependencies {
-  import play.core.PlayVersion
 
-  val compile: Seq[ModuleID] = Seq(
+  private lazy val compile = Seq(
     play.sbt.PlayImport.ws,
-    "org.reactivemongo" %% "play2-reactivemongo"            % "0.20.13-play28",
-    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "1.1.0-play-28",
-    "uk.gov.hmrc"       %% "domain"                         % "8.1.0-play-28",
-    "com.typesafe.play" %% "play-json-joda"                 % "2.7.4",
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.11.0-play-28",
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "5.24.0"
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"             % "0.73.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"             % "3.32.0-play-28",
+    "uk.gov.hmrc"             %% "domain"                         % "8.1.0-play-28",
+    "uk.gov.hmrc"             %% "play-conditional-form-mapping"  % "1.12.0-play-28",
+    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28"     % "7.8.0"
   )
 
-  val test: Seq[ModuleID] = Seq(
-    "org.scalatest"               %% "scalatest"             % "3.2.12",
-    "org.scalatestplus.play"      %% "scalatestplus-play"    % "5.1.0",
-    "org.pegdown"                 %  "pegdown"               % "1.6.0",
-    "org.jsoup"                   %  "jsoup"                 % "1.15.1",
-    "com.typesafe.play"           %% "play-test"             % PlayVersion.current,
-    "wolfendale"                  %% "scalacheck-gen-regexp" % "0.1.2",
-    "org.mockito"                 %  "mockito-all"           % "1.10.19",
-    "org.mockito"                 % "mockito-core"           % "4.6.1",
-    "org.scalatestplus"           %% "mockito-3-12"          % "3.2.10.0",
+  private lazy val test: Seq[ModuleID] = Seq(
+    "com.typesafe.play"           %% "play-test"                % PlayVersion.current,
+    "org.scalatestplus.play"      %% "scalatestplus-play"       % "5.1.0",
+    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-28"  % "0.73.0",
     "org.scalatestplus"           %% "scalatestplus-scalacheck" % "3.1.0.0-RC2",
-    "org.scalacheck"              %% "scalacheck"            % "1.16.0",
-    "com.github.tomakehurst"      % "wiremock-standalone"    % "2.27.2",
-    "com.vladsch.flexmark"        % "flexmark-all"           % "0.62.2"
+    "org.jsoup"                   %  "jsoup"                    % "1.15.3",
+    "org.scalatest"               %% "scalatest"                % "3.2.14",
+    "org.scalatestplus"           %% "mockito-4-6"              % "3.2.14.0",
```

Comment:
This can be replaced by "org.mockito"   %% "mockito-scala-scalatest" % "1.17.12"

Reply:
I dont see any reason why to do that, could you elaborate why you decided this?

Reply:
I think its not best approach to stick to a single version "mockito-4-6" mainly when there is an option to be replaced

Reply:
done good change 👍